monitoring/README.md

```diff
@@ -3,44 +3,101 @@
 <img src="images/ks-monitoring-setup.png" width="80%" height="80%" title="kubestellar-monitoring-setup">
 
 
+### Requirement
+To follow the instructions here, you need to have release `v0.2.0-rc12` or higher for the status-addon controller and agent installed in your enviroment. If you're using an older release, please follow these instructions to update your environment: 
```

Comment:
Rather than do a special hack here, we should first follow the instructions in https://docs.kubestellar.io/release-0.24.0/direct/release/#reacting-to-a-new-ocm-status-addon-release . After that, no special hackery would be needed here.

Reply:
This is just a temporary hack until we have a new release for KS.

Reply:
But someone using `main` no longer needs this hack, right? (#2468 has merged.)

Reply:
To address that - I updated the README file with the following sentence: 
```
To follow the instructions here you need to use KubeStellar main branch or any future KubeStellar release higher than 0.24.0
```
I also created this issue to keep track of improvements needed in the instructions: https://github.com/kubestellar/kubestellar/issues/2470